### DISCUSSION TITLE
Clarifying questions from a first time laser-cutter / pcb printer

### DISCUSSION THREAD

**Discussion by KarimShoorbajee** [FundamentalFrequency/.github#21] [Q&A] [answered]
This is super cool and I'm looking to try and build this very soon but I had some questions

1. For my own understanding's sake why do we need to 6 of the lower layer of the case? Could we not just cut one thick layer? Is it just easier to cut that way? Or maybe I'm missing a part of the instructions?
2. I was looking to order the pcb and case parts to be printed online. I was eyeing [jlcpcb](https://jlcpcb.com/) for the pcb and [sendcutsend](https://sendcutsend.com/) for the case. Any thoughts or alternatives? I know there's a minimum quantity of pcb boards that I'd need to order so I'd be happy to send my spares to others who would like to build it (or participate in a group buy).
3. For flashing the teensy firmware: I don't have a machine that runs linux natively. Would I run into any issues doing this set up with a virtual box (linux vm) running on windows? Or would I be better off booting linux?

Greatly appreciate any clarification/guidance!

**Comment by stonepreston** [comment, marked as answer]
1. Acrylic is usually sold in thin sheets. You can do less layers if you can find a thicker acrylic, but it probably wouldnt make much difference. eighth inch acrylic is pretty standard. So thats why the layers are 3 mm. 

2. JLC is what  I used for the PCB. I have never used a laser cutting service before so cant offer much advice there. Someone else mentioned sendcutsend though. Another one ive seen is ponoko.

3. You could flash on windows if you wanted provided you can compile the firmware. Its probably easier to use a VM I dont think that will give you any problems.

**Reply by KarimShoorbajee** [reply]
Ah nice. Thanks for the quick reply. Very excited about this!

**Reply by stonepreston** [reply]
@KarimShoorbajee happy to help!
